frontend/src/index.js

```diff
@@ -1,14 +1,17 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
+import { BrowserRouter as Router } from "react-router-dom";
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Router>
```

Comment:
Jó lesz, de törik a build ezzel: Invariant failed: You should not use <Switch> outside a <Router>
Ez azért van, mert van egy teszt az App.js-re, ami elhal, mert a Switch-nek mindig a Routerben kell lennie. Ha csak az app komponenst használom, akkor ott nincs a Switch körül Router, a Routert mindig a Switch-et definiáló komponensbe rakjuk ezért, tehát ha eggyel beljebb mozgatod az App komponensbe a Routert, akkor jó lesz.